features/270-next-skycli/api.md

```diff
@@ -64,10 +64,10 @@ skycli [group] [verb/action] [parameters]
     - [skycli app invoke-function [FUNCTION_NAME] --payload [PAYLOAD_JSON]](commands.md#skycli-app-invoke-functions)
 
 - skycli domain
-    - [skycli domain add [CUSTOM_DOMAIN]](commands.md#skycli-domain-create)
+    - [skycli domain add [CUSTOM_DOMAIN]](commands.md#skycli-domain-add)
```

Comment:
Is there no way to delete the domain, except removing DNS record and let it naturally expire?

Reply:
Let me add the domain remove command :)